src/styles/3-UI/_card-button.scss

```diff
@@ -0,0 +1,24 @@
+.card-button {
+  display: block;
+  border-radius: 5px;
+  background-color: $button-background;
+  padding-top: 10.2px;
+  padding-bottom: 10.2px;
```

Comment:
Avoid float numbers for pixels. Also, you can use shorthand property padding instead.